daffodil-test/src/test/resources/org/apache/daffodil/section00/general/disallowDocTypes.tdml

```diff
@@ -59,8 +59,6 @@
     <tdml:document/>
     <tdml:errors>
       <tdml:error>Schema Definition Error</tdml:error>
-      <tdml:error>org.xml.sax.SAXParseException</tdml:error>
-      <tdml:error>hasDocType-include.dfdl.xsd</tdml:error>
       <tdml:error>DOCTYPE is disallowed</tdml:error>
```

Comment:
hasDocType-include.dfdl.xsd looks like a diagnosticPath. Did we lose that?

Reply:
It's actually the systemId from the XercesError which we lost when we converted it to a schemaFileLocation since we don't grab the absolute system id from the Xerces error and use the diagnosticFilepath from the schemaFileLocation passed into the errorhandler instead